test/Makefile

```diff
@@ -43,7 +43,7 @@ endif
 SRC_DIR=../src
 
 TEST_DIR=.
-TEST_SOURCES=$(wildcard $(TEST_DIR)/*_unittest.cc)
+TEST_SOURCES=$(wildcard $(TEST_DIR)/*_unittest.cc $(TEST_DIR)/fake_clock.cc $(TEST_DIR)/fake_http_server.cc)
```

Comment:
How about just `$(TEST_DIR)/*.cc`?

Reply:
Done.

Reply:
What is the benefit of explicitly listing every file? It feels like this way we have to modify the Makefile every time we add a new unit test file / utility file. Can't we rely on the pattern match (*_unittest, *.cc)?

Reply:
Sync'ed offline, the refactoring to use wildcard will happen in a separate PR.